docs/reference/schema-viewer-navigation.md

```diff
@@ -0,0 +1,169 @@
+---
+id: schema-viewer-navigation
+title: Schema Viewer Usage
+---
+
+In this section, we will guide you on navigating the [Schema Viewer](https://smartdatalake.ch/json-schema-viewer/), a powerful tool designed to help you understand the structure and relationships of the SDLB API. Additionally, we will delve into the process of mapping the schema logic to the configuration file, allowing you to effectively write SDLB pipelines.
```

Comment:
Could you please describe a little in more detail what the aim of the schema viewer is and how and what we use it for. Speeking we search and identify dataObjects... we create the SDLB config with. Also providing a full list od different types etc.